setup.py

```diff
@@ -14,7 +14,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.0.3',
+    version='1.1.0',
```

Comment:
I'd prefer we got the package version from the code itself (easiest way is method 6 at https://packaging.python.org/en/latest/single_source_version/#single-sourcing-the-project-version). For example:

```
# resin-sdk-python/resin/__init__.py

__version__ = '1.1.0'

# blah code...
```

And then in ``setup.py``:

```
import resin

setup(
  ...
  version=resin.__version__,
  ...
)
```